packages/lib/package.json

```diff
@@ -62,6 +62,7 @@
 		"eslint-plugin-jest": "^26.1.3",
 		"jest": "^27.5.1",
 		"jest-extended": "^0.11.5",
+		"log4js": "6.3.0",
```

Comment:
I would have expected `@log4js-node/log4js-api` as a regular dependency `log4js` as a dependency only in the cli itself. Is there something I'm missing?